vite.config.js

```diff
@@ -1,12 +1,15 @@
-import { sentrySvelteKit } from "@sentry/sveltekit";
+import { sentrySvelteKit } from '@sentry/sveltekit';
```

Comment:
Since we've removed sentry from the package.json we also need to remove it from the Vite config
```suggestion
```